dependencies.gradle

```diff
@@ -28,6 +28,7 @@ ext {
     revCuratorTest = '2.4.0'
     revDynoQueues = '2.0.20'
     revElasticSearch6 = '6.8.12'
+    revElasticSearch7 = '7.10.1'
```

Comment:
We try our best to align with Spring managed versions of dependencies. The version that spring boot [2.3.12.RELEASE](https://docs.spring.io/spring-boot/docs/2.3.12.RELEASE/reference/html/dependency-versions.html) recommends for elastic search is `7.6.2`. Can you update the version here to match that?

Reply:
updated